src/Build.UnitTests/BackEnd/TaskBuilder_Tests.cs

```diff
@@ -1098,7 +1086,6 @@ private Project CreateSTATestProject(bool requireSTA, bool failTask, bool throwE
 
             return project;
         }
-#endif
```

Comment:
this is accidentally deleted, i think

Reply:
oh, I see. it is moved to the place after `GenerateSTATask` which is used in this block